__tests__/data/sanity/active_campaign_output.json

```diff
@@ -0,0 +1,3 @@
+{
+
```

Comment:
Under the sanity folder, we will have each of the sanity output files for each of the destinations.